ChartsRealm.podspec

```diff
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = "10.11"
   s.source = { :git => "https://github.com/danielgindi/ChartsRealm.git", :tag => "v#{s.version}" }
   s.source_files  = "ChartsRealm/Classes/**/*.swift"
-  s.dependency "Charts", "3.0.3"
+  s.dependency "Charts", "3.0.4"
   s.dependency "RealmSwift", "~> 2.10"
```

Comment:
shall we bump to 2.10.2 as well?

Reply:
No. We should leave it. All versions of 2.10 should be API compatible.

Reply:
sure. good to go then

Reply:
making sure everything still builds and the pod validates then ill merge

Reply:
sure. good to go then. My carthage update always timeout after a long time downloading realm swift

Reply:
You should use the build command when testing. It's `carthage build --no-skip-current`